src/main/java/io/confluent/idesidecar/restapi/application/SidecarInfo.java

```diff
@@ -67,6 +67,7 @@ static String semanticVersionWithin(Properties props, String key, String def) {
 
   static final String OS_ARCH_KEY = "os.arch";
   static final String OS_NAME_KEY = "os.name";
+  static final String OS_TYPE_KEY = "os.type";
```

Comment:
`os.type` is not a real System property. We coopted the name `osType()` to be similar to `osName()` except that it returns one of the `OperatingSystemType` literals. See line #99 in this class.
```suggestion
```